### ISSUE TITLE
Other projection/coordinate system support other than WGS84 lat/lon

### ISSUE THREAD

**Issue by wonderchook** [Cadasta/cadasta-platform#43]
As a user I want to be able to upload data represented/captured in coordinate systems other than WGS-84

requested by @nastynoel and moved from trello

**Comment by oliverroick** [comment]
That sounds like a case for GDAL; there are Python bindings we can use.

**Comment by wonderchook** [comment]
yes, agreed. I'm not quite sure where this fits in right now. Perhaps when uploading existing data?

**Comment by wonderchook** [comment]
should be part of importing data

**Comment by wonderchook** [comment]
@dpalomino we need to gather more specific requirements for this. 

@bjohare do you think it matters if we tackle this after the rest of the data import stuff has been done?

**Comment by bjohare** [comment]
@wonderchook we should be able to retrofit this to the data import work. Some additional requirements would be useful though..

**Comment by wonderchook** [comment]
Assigning to @dpalomino for requirements development so it can be included in Sprint 10.

**Comment by dpalomino** [comment]
Thanks @wonderchook, I will create some requirements for this.
